models/migrations/v1_22/v292.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2024 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package v1_22 //nolint
+
+import (
+	"code.gitea.io/gitea/models/project"
+
+	"xorm.io/xorm"
+)
+
+func AddDefaultProjectColumn(x *xorm.Engine) error {
+	var projectsWithMissingDefault []project.Project
+	x.SQL("SELECT DISTINCT `p`.`id`, `p`.`creator_id` FROM `project` `p` WHERE NOT EXISTS (SELECT 1 FROM `project_board` `pb` WHERE `pb`.`project_id` = `p`.`id` AND `pb`.`default` != 0);").
```

Comment:
```suggestion
	x.SQL("SELECT DISTINCT `p`.`id`, `p`.`creator_id` FROM `project` `p` WHERE NOT EXISTS (SELECT 1 FROM `project_board` `pb` WHERE `pb`.`project_id` = `p`.`id` AND `pb`.`default` != 0)").
```

Semicolon not needed. It is a CLI-only thing in SQL.